File: Bijele/bijele.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]) {
    int king, queen, rooks, bishops, knights, pawns;
    scanf("%d %d %d %d %d %d", &king, &queen, &rooks, &bishops, &knights, &pawns);
    printf("%d %d %d %d %d %d\n", 1 - king, 1 - queen, 2 - rooks, 2 - bishops,2 - knights, 8 - pawns);
    return EXIT_SUCCESS;
}

File: python-solutions/Bijele/bijele.py
import sys

king = 1
queen = 1
rooks = 2
bishops = 2
knights = 2
pawns = 8
kingInput = int(input())
queenInput = int(input())
rooksInput = int(input())
bishopsInput = int(input())
knightsInput = int(input())
pawnsInput = int(input())
print(king - kingInput,queen - queenInput,rooks - rooksInput,bishops - bishopsInput,knights - knightsInput,pawns - pawnsInput)
sys.exit(0)
